src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1286,9 +1286,13 @@
                           (utils/pem->cert)
                           (utils/get-serial))
                      subjects)
+        serial-count (count serials)
         new-crl (utils/revoke-multiple original-crl ca-private-key
                                        ca-public-key serials)]
-    (utils/crl->pem! new-crl cacrl)))
+    (utils/crl->pem! new-crl cacrl)
+    (log/debug (if (= 1 serial-count)
+                 (i18n/trs "Revoked 1 certificate.")
```

Comment:
doh

Reply:
I don't see anything?

Reply:
Sorry, I was commenting about how that was a fixup to the version in 6.x.

Reply:
Also, unrelated to my comment, we should have made this change a separate commit in 6.x so we could cherry-pick it back.